#播放通知消息

int32   speak_state		     #播放状态通知，0：播放开始 1：播放结束
